### PR TITLE
Microsoft Graph and NextJs workshops

### PR DESCRIPTION
**Pull request by glaucia86**
Hi, @DanWahlin and @aycabas!

I am sending the workshops recently created it. Please, any question reach me out anytime.

### CODE REVIEW

File: tutorials/docs/Take-A-Break-Reminder-App/01-Create-An-Application-Azure-Active-Directory.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+---
+title: Session 02 - Configuring an Application in Azure Active Directory
+sidebar_position: 1
+---
+
+> **Note**: If you are coming from the previous workshop, you can skip this session.
```
Comment (aycabas) [inline tutorials/docs/Take-A-Break-Reminder-App/01-Create-An-Application-Azure-Active-Directory.md:6]: @glaucia86 previous workshop includes different setup and app permissions in AAD app. I would remove this note and let developers go through this step even if they completed the previous workshop.

File: tutorials/docs/Take-A-Break-Reminder-App/02-Installing-Dependencies-From-Kit-Get-Started.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+title: Session 03 - Installing dependencies from the Kit Get Started
+sidebar_position: 1
+---
+
+Again, if you are not coming from the previous workshop, you can download the source code of the project we will be working on in this workshop **[HERE](https://github.com/glaucia86/kitstarter-msgraph-nextjs)**
```
Comment (aycabas) [inline tutorials/docs/Take-A-Break-Reminder-App/02-Installing-Dependencies-From-Kit-Get-Started.md:6]: @glaucia86 again I would keep the workshops seperate and make sure that everyone downloaded the source code even if they did the same in the previous workshop.

**Comment by glaucia86** [comment]
@aycabas and @DanWahlin merged conflicts

**Review by DanWahlin — CHANGES_REQUESTED** [review]
This looks great Glaucia! Really nice job on putting this together.

A few minor things to adjust for consistency:

1. Would it be possible to take out "Session XX" in the TOC and put the number instead? For example, instead of "Session 02 - Configuring an Application in Azure Active Directory", it'd be "1. Configuring an Application in Azure Directory". That way they all look the same across the tutorials/workshops. You'll see an example of what I mean with this tutorial: http://localhost:3000/MicrosoftCloud/tutorials/docs/ACS-to-Teams-Meeting/
2. Very minor, but could you rename image-101.PNG to image-101.png? It gives a build warning on Linux due to a case sensitivity issue.
3. All of the tutorials on the site are technically workshops so let's go with "Authentication App with Next.js and Microsoft Graph" as the title rather than "Workshop: Authentication App with Next.js and Microsoft Graph" just for the sake of consistency. 
4. We're using "Exercise X" as the main title for each markdown file. We then have the actual title right below that. If you'd prefer to use "Session XX" and leave what you have in the markdown files now that's fine though. If you want to change to our format that's fine too. Totally your call on that one. Example of what we're doing here:

https://raw.githubusercontent.com/microsoft/MicrosoftCloud/main/tutorials/docs/ACS-to-Teams-Meeting/01-Create-ACS-Resource.md

Other than those quick changes, I think it looks great (didn't have time to go through it from a technical standpoint - but from a formatting standpoint it looks great). I like the friendly tone you use in the parts I did read through. :-)  Thanks again!

**Comment by glaucia86** [comment]
> This looks great Glaucia! Really nice job on putting this together.
> 
> A few minor things to adjust for consistency:
> 
> 1. Would it be possible to take out "Session XX" in the TOC and put the number instead? For example, instead of "Session 02 - Configuring an Application in Azure Active Directory", it'd be "1. Configuring an Application in Azure Directory". That way they all look the same across the tutorials/workshops. You'll see an example of what I mean with this tutorial: http://localhost:3000/MicrosoftCloud/tutorials/docs/ACS-to-Teams-Meeting/

Done!

**Comment by glaucia86** [comment]
> This looks great Glaucia! Really nice job on putting this together.
> 
> A few minor things to adjust for consistency:
> 

> 2. Very minor, but could you rename image-101.PNG to image-101.png? It gives a build warning on Linux due to a case sensitivity issue.

Done!

**Comment by DanWahlin** [comment]
Thanks! Really appreciate the contribution.

**Pull request merged**